src/hooks/index.ts

```diff
@@ -794,3 +827,29 @@ export const useResync = (wallet: YoroiWallet, options?: UseMutationOptions<void
     resync: mutation.mutate,
   }
 }
+
+export const useNfts = (wallet: YoroiWallet, options: UseQueryOptions<YoroiNft[], Error> = {}) => {
+  const {data, refetch, ...rest} = useQuery({
+    queryKey: [wallet.id, 'nfts'],
+    queryFn: () => wallet.fetchNfts(),
+    refetchOnMount: false,
+    refetchIntervalInBackground: false,
+    refetchInterval: false,
+    refetchOnWindowFocus: false,
+    refetchOnReconnect: false,
+    ...options,
```

Comment:
...options should come first so it doesn't clobber

Reply:
in this case move queryKey/queryFn after `...options,`